### PR TITLE
{vis}[intel/2015b] OpenCV 2.4.9 (WIP)

### PR DESCRIPTION
**Pull request by valtandor**
Easyconfig for OpenCV 2.4.9 with the intel-2015a toolchain

### CODE REVIEW

**Comment by valtandor** [comment]
Yes

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/o/OpenCV/OpenCV-2.4.9-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+python = 'Python'
+pythonver = '2.7.9'
+pythonshortver = '.'.join(pythonver.split('.')[0:2])
+java = 'Java'
+javaver = '1.7.0_51'
+
+dependencies = [
+    (python, pythonver),
+    ('CMake', '3.0.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenCV/OpenCV-2.4.9-intel-2015a.eb]: CMake is only a build dependency, and his is a rather old version?

File: easybuild/easyconfigs/o/OpenCV/OpenCV-2.4.9-intel-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+]
+
+preconfigopts = 'export IPPROOT=$EBROOTICC/ipp && '
+
+configopts = '-DCMAKE_BUILD_TYPE=RELEASE '
+configopts += '-DBUILD_PYTHON_SUPPORT=ON '
+configopts += '-DPYTHON_PACKAGES_PATH=$EBROOTPYTHON/lib/python%s/site-packages ' % pythonshortver
+configopts += '-DPYTHON_EXECUTABLE=$EBROOTPYTHON/bin/python '
+configopts += '-DPYTHON_LIBRARIES=$EBROOTPYTHON/lib '
+configopts += '-DPYTHON_INCLUDE_PATH=$EBROOTPYTHON/include/python%s ' % pythonshortver
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenCV/OpenCV-2.4.9-intel-2015a.eb]: these `-DPYTHON_*` are new, they should be added to the other OpenCV easyconfigs too?

File: easybuild/easyconfigs/o/OpenCV/OpenCV-2.4.9-intel-2015a.eb [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    (python, pythonver),
+    ('CMake', '3.0.0'),
+    ('zlib', '1.2.8'),
+    ('ffmpeg', '2.4'),
+    ('libjpeg-turbo', '1.3.1'),
+    ('libpng', '1.6.12'),
+    ('LibTIFF', '4.0.3'),
+    ('JasPer', '1.900.1'),
+    (java, javaver ,'', True),
+    ('ant', '1.9.3', '-%s-%s' % (java,javaver), True),
```
Comment (boegel) [inline easybuild/easyconfigs/o/OpenCV/OpenCV-2.4.9-intel-2015a.eb]: space after `,`

**Comment by boegel** [comment]
@valtandor: open remarks, and Jenkins reports:

rresolvable dependencies encountered: ffmpeg/2.4-intel-2015a, libpng/1.6.12-intel-2015a, Java/1.7.0_51, ant/1.9.3-Java-1.7.0_51'

**Comment by valtandor** [comment]
@boegel: Indeed. A new test cycle was only triggered because I merged some recent changes in from the "develop" branch and pushed them to my branch on GitHub. I've been meaning to have a look at this one, along with the colleague who first installed this package on our cluster. Thanks for the note re dependencies.

**Comment by boegel** [comment]
@valtandor: I've marked this as WIP in the PR title, feel free to change it back once you feel this is finished

**Comment by valtandor** [comment]
@boegel @jordiblasco Like MIGRATE, I've decided not to pursue this for the time being.

**Pull request closed without merging**